projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/usage-bar/index.tsx

```diff
@@ -49,17 +50,37 @@ function UsageControl( {
 	hasFeature,
 	requestsCount,
 	requestsLimit,
-}: Pick< AIFeatureProps, 'isOverLimit' | 'hasFeature' | 'requestsCount' | 'requestsLimit' > ) {
+	usagePeriod,
+}: Pick<
+	AIFeatureProps,
+	'isOverLimit' | 'hasFeature' | 'requestsCount' | 'requestsLimit' | 'usagePeriod'
+> ) {
+	// Compute the number of days from start to now
+	let resetMsg = '';
+	if ( usagePeriod?.currentStart ) {
```

Comment:
```suggestion
	if ( usagePeriod?.nextStart ) {
```

I think we need to check for the next period here, since it's the one being used on the code.